tests/python/PyUnusedCodeBearTest.py

```diff
@@ -1,19 +1,33 @@
-from queue import Queue
-
 from bears.python.PyUnusedCodeBear import PyUnusedCodeBear
-from tests.LocalBearTestHelper import LocalBearTestHelper
-from coalib.settings.Section import Section
+from tests.LocalBearTestHelper import verify_local_bear
 
+test_file1 = """
```

Comment:
Use proper names. Don't enumerate. Your variable name should tell if its valid or not and why.

example: invalid_unused_imports